wiki/Community/osu!_community_meetings/en.md

```diff
@@ -8,7 +8,7 @@ Currently, the osu! community meetings are being chaired by [peppy](https://osu.
 
 Meetings are generally held every two weeks, with reminders announced on [peppy's Twitter](https://twitter.com/ppy) about a day beforehand.
 
-The next osu! community meeting is scheduled to take place on: **February 20, 2022 (Sunday 07:00 UTC)**.
+The next osu! community meeting date will be announced at a later date.
```

Comment:
I'm not really sure it's useful to include this and have to constantly update. Maybe just bold the "every two weeks" part above?